src/main/java/com/hoverla/bring/exception/InitializePropertyException.java

```diff
@@ -0,0 +1,7 @@
+package com.hoverla.bring.exception;
+
+public class InitializePropertyException extends RuntimeException {
```

Comment:
I think, we should create base Exception and enxtend from this exception. What do you think?